tests/Tokenizer/TokensTest.php

```diff
@@ -436,9 +436,9 @@ public static function provideMonolithicPhpDetectionCases(): iterable
         yield [false, "<?=' '\n?><?\n"];
 
         // short open tag echo
-        yield [true, "<?=' ';\n"];
+        yield [false, "<?=' ';\n"];
 
-        yield [true, "<?=' '?>"];
+        yield [false, "<?=' '?>"];
```

Comment:
Why these cases have been changed? Both look like a monolithic PHP (`<?php echo ' ';` with trailing new line, `<?php echo ' '; ?>`). In general such changes should be considered as BC break.

Reply:
In almost all fixers the `Tokens::isMonolithicPhp()` was/is further checked with `T_OPEN_TAG` only, ie. without `T_OPEN_TAG_WITH_ECHO`. I belive this change is thus good unification and I do not expect any CI to be broken - we no longer fix exotic `T_OPEN_TAG_WITH_ECHO`, thus fixes code before will be kept green in CI.

Reply:
FYI: recently there was a [fix for short echo syntax](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7637), so yeah, even if there are not many fixers that handle this, I believe we should not change the contract just like that. Maybe you shouldn't touch more fixers than `declare_strict_types` 🤔?